app/services/assign_edition_status_service.rb

```diff
@@ -18,6 +18,7 @@ def call
     if update_last_edited
       edition.last_edited_by = user
       edition.last_edited_at = Time.current
+      edition.add_edition_editor(user) if user
```

Comment:
Is the `if user` guard clause needed here?  We're referencing `user` several times elsewhere in this method without checking whether it is defined.

Reply:
The guard is needed, because sometimes the `AssignEditionStatusService` is called without passing in a user, but I'm going to move it to the `add_edition_editor` method.

Reply:
Not a biggie, but we planned to have this guard here as it didn't really make sense to have the method of add_editor accept a nil argument.